### ISSUE TITLE
Stats and Pages Quick Actions are not shown on Dashboard after creating a new site

### ISSUE THREAD

**Issue by staskus** [wordpress-mobile/WordPress-iOS#22440]
### Expected behavior

Quick actions such as Stats, and Pages should be shown as soon as they are available.

### Actual behavior

After creating a site only Posts and Media are shown on Dashboard. 

**Workaround:** Only after relaunching the app do more options show up.

https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/c0af323b-2abe-4afc-9a99-fc45ea63b747

### Steps to reproduce the behavior

1. Open the Jetpack app
2. Login to an account with no sites
3. Create a site
4. Notice only Posts and Media Quick Actions available
5. Click more, and confirm Pages and Stats can be accessed
6. Come back to the Dashboard, and refresh it
7. Confirm that only Posts and Media Quick Actions are available
8. Close and relaunch the app
9. Notice that all Quick Actions, including Stats and Pages, are now visible

**Comment by jhmonroe** [comment]
Confirming this bug occurs in the order you said!

If a user logs into the app before adding the site, once added, the site will not show PAGES menu item (but posts and media shows).
If user installs the app fresh, then ADD SITE, then login... PAGES menu item shows up

Logging into the app but not adding a site first seems to create a bit of a limbo situation! :-)

**Comment by kean** [comment]
Duplicate of https://github.com/wordpress-mobile/WordPress-iOS/issues/22440
